sensor_msgs/LaserScan scan
geometry_msgs/Pose pose
InterestPointRos[] pts
